### ISSUE TITLE
How to get the loss of a model?

### ISSUE THREAD

**Issue by jordan-bird** [Kyubyong/dc_tts#43]
Hi,

Sorry, I'm new to TensorFlow. How can I output a graph of the loss over time? 

Is it possible to also get the best loss in order to compare two different models after n iterations? I want to train two tts models on the same dataset and compare the two

thanks!

**Comment by jordan-bird** [comment]
PS: I would also like to train two differently formatted datasets with dc_tts and then compare how they fare - ie. using two different kinds of markup in each otherwise identical dataset

**Comment by zabir-nabil** [comment]
Just run tensorboard from logdir folder, with `tensorboard --logdir=LJ01-1`. Go to browser and type http://localhost:6006/ and search for loss.
